network-api/networkapi/wagtailpages/migrations/0004_auto_20200713_1757.py

```diff
@@ -0,0 +1,175 @@
+# Generated by Django 2.2.13 on 2020-07-13 17:57
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailimages', '0022_uploadedimage'),
```

Comment:
I'm using a copy of the staging database. This `('wagtailimages', '0022_uploadedimage'),` causes Django to complain. But if commented out I can migrate fine. 

@Pomax have you run into something like this before?

Reply:
When I run this from `inv new-db` (a new database), delete the original migration and re-create this one I get a different dependency. 

```
    dependencies = [
        ('wagtailimages', '0001_squashed_0021'),  # <- This one
        ('wagtailpages', '0003_delete_homepagefeaturednews'),
    ]
```

Reply:
Yeah, let's recreate this migration with a new `inv new-db` on local is possible, please and thanks!

Reply:
For sure. I probably missed something in between... been fixing my migration file too much. Will update PR with a new migration file and see if it fixes the issue.

Reply:
> When the quote is short, should the entire text area (to the right of the image) be vertically centered?
I don't think we need that. It is good as is (I don't think the effort is worth it). Plus it is very unlikely the quote will be that short, anyways. Thanks for checking!

Reply:
@KalobTaulien I followed the steps you mentioned but the new migration file I got still has dependency `('wagtailimages', '0022_uploadedimage')`. 🤔 I will ask Pomax to try it tomorrow.

Reply:
@mmmavis I'll re-test this. My docker was acting up and I didnt realize it. It's likely this issue is on my end, not yours.

Reply:
Yep, scratch this. This was my environment acting up. Sorry for the confusion folks!

Reply:
I can't reproduce this problem. Steps taken:

- ensure clean slate: `docker system prune --all`
- get/update branch: `git fetch origin & git checkout issue-4843-quote & git pull origin issue-4843-quote`
- build docker environment: `inv new-env`
- clone staging db: `docker-compose up postgres` followed by a staging db copy into the local db
- apply the branch migrations: `inv migrate`:

```bash
>inv migrate
cd C:\Users\Mike\Documents\Git\mozilla\foundation.mozilla.org && docker-compose run --rm backend ./dockerpythonvenv/bin/python network-api/manage.py migrate --n
o-input
Starting foundationmozillaorg_watch-static-files_1 ...
Starting foundationmozillaorg_postgres_1           ...
Operations to perform:org_postgres_1           ... done                                                                                                    sites, sociaOperations to perform:org_postgres_1           ... done                                                                                                    sites, socia  Apply all migrations: admin, auth, buyersguide, campaign, contenttypes, experiments, highlights, milestones, mozfest, news, people, redirects, sessions, sites, social_django, taggit, wagtailadmin, wagtailcore, wagtaildocs, wagtailembeds, wagtailforms, wagtailimages, wagtailinventory, wagtailmetadata, wagtailpages, wagtailredirects, wagtailsearch, wagtailusers
Running migrations:
  Applying wagtailpages.0004_auto_20200716_1708... OK
No new translatable fields detected
```
- then run all the things using `docker-compose up`.

Following these steps, things work as expected. Note that the wagtail version that I end up with 2.9.2 which includes that `0022_uploadedimage` as latest `wagtail.images ` migration. If you're seeing `0001_squashed_0021` @KalobTaulien it is entirely possible you're getting an older version of wagtail for some reason.

Reply:
Yay glad we found out what the issue was. Thanks for the thorough test @KalobTaulien and @Pomax for chiming in.